api/pyproject.toml

```diff
@@ -117,7 +117,7 @@ beautifulsoup4 = "4.12.2"
 boto3 = "1.35.17"
 bs4 = "~0.0.1"
 cachetools = "~5.3.0"
-celery = "~5.3.6"
+celery = "~5.4.0"
```

Comment:
Is the worker logging a new feature in 5.4.x? Any doc reference for the reason  bumping celery version ?

Reply:
The worker_logfile is a new feature added in Celery 5.4, which allows the worker logs to be output to a specified log. Please see the attached file for details.

Reply:
[https://docs.celeryq.dev/en/stable/userguide/configuration.html](url)